tailwind.base.css

```diff
@@ -132,12 +136,12 @@
   /* BADGE CLASS */
   .gn-ui-badge {
     --rounded: var(--gn-ui-badge-rounded, 0.25em);
-    --padding: var(--gn-ui-badge-padding, 0.375em 0.75em);
+    --padding: var(--gn-ui-badge-padding, 0.375em 0.875em);
     --text-color: var(--gn-ui-badge-text-color, var(--color-gray-50));
     --background-color: var(--gn-ui-badge-background-color, black);
     --opacity: var(--gn-ui-badge-opacity, 0.7);
-    @apply inline-block opacity-[--opacity] p-[--padding] rounded-[--rounded]
-      font-medium text-[length:0.875em] leading-none text-[color:--text-color] bg-[color:--background-color];
+    @apply opacity-[--opacity] p-[--padding] rounded-[--rounded]
+      font-medium text-[length:0.875em] text-[color:--text-color] text-xs bg-[color:--background-color] flex justify-center items-center content-center;
```

Comment:
We should keep in mind that these changes affect all badges in gn-ui including the datahub.

Reply:
Yes, I checked the datahub and badges seems ok.